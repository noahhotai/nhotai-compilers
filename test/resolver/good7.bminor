main: function integer (x: integer, y: integer, z: integer) = {

    x++;
    y: integer = x;
    z--;
}