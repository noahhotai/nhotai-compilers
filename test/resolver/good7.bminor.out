Program parsed successfully.
x resolves to param 0
x resolves to param 0
z resolves to param 2
Program resolved successfully.
